### PR TITLE
Add EmbedLiteView::Repaint() API

### PR DESCRIPTION
**Pull request by rojkov**
in order to repaint recreated GL buffer.

I'm not sure the code of EmbedLiteViewThreadChild::RecvRepaint() is optimal, but it works.

### CODE REVIEW

File: embedding/embedlite/embedthread/EmbedLiteViewThreadParent.cpp [hunk 1]
```diff
@@ -426,6 +426,14 @@ EmbedLiteViewThreadParent::SuspendRendering()
 }
 
 NS_IMETHODIMP
+EmbedLiteViewThreadParent::Repaint()
+{
+
+  unused << SendRepaint();
```
Comment (tmeshkova) [inline embedding/embedlite/embedthread/EmbedLiteViewThreadParent.cpp]: I think it is better to call EmbedLiteCompositorParent::ResumeRendering, and call ScheduleRenderOnCompositorThread(); in that method

**Comment by rojkov** [comment]
I tried to put ScheduleRenderOnCompositorThread() to EmbedLiteCompositorParent::ResumeRendering and called SuspendRendering and ResumeRendering from QuickMozView::suspendView() and QuickMozView::resumeView(), but it didn't work

**Comment by rojkov** [comment]
I've dropped EmbedLiteViewThreadChild::RecvRepaint().

**Comment by rojkov** [comment]
Will resubmit

**Pull request closed without merging**